infrastructure/environments/staging/services/api-deployment.yml

```diff
@@ -17,28 +17,31 @@ spec:
           imagePullPolicy: Always
           name: main
           env:
-            - name: NODE_ENV
-              value: "staging"
-            - name: MONGO_URI
-              value: "mongodb://mongo:27017/bedrock_staging"
+            # - name: NODE_ENV
```

Comment:
is the intent to remove these lines eventually?

Reply:
Yes, those lines will be removed. I left them for illustrative purposes.